neon_audio/utils.py

```diff
@@ -62,7 +63,18 @@ def _plugin_to_package(plugin: str) -> str:
     }
     return known_plugins.get(plugin) or plugin
 
+def build_extra_dependency_list(config: Union[dict, Configuration], additional: List[str] = []) -> str:
+    extra_dependencies = config.get("extra_dependencies", {})
+    dependencies = additional + extra_dependencies.get("global", []) + extra_dependencies.get("neon-audio", [])
```

Comment:
Please refactor `neon-audio` to `audio` to match the modules used in [`ready_settings`](https://github.com/OpenVoiceOS/ovos-config/blob/e622d44a5e3fc259e54b50f85e7001e70e7eb8f6/ovos_config/mycroft.conf#L112-L121)